.github/workflows/test.yml

```diff
@@ -33,16 +33,23 @@ jobs:
           keyword: "[test-upstream]"
 
   test:
-    name: "Build & Test (${{ matrix.os }}, python: ${{ matrix.python }})"
+    name: "Build & Test (${{ matrix.os }}, python: ${{ matrix.python }}, distributed: ${{ matrix.distributed }})"
     needs: [detect-ci-trigger]
     runs-on: ${{ matrix.os }}
     env:
       CONDA_FILE: continuous_integration/environment-${{ matrix.python }}-dev.yaml
+      DASK_SQL_DISTRIBUTED_TESTS: ${{ matrix.distributed }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         python: ["3.8", "3.9", "3.10"]
+        distributed: [false]
+        include:
+          # run tests on a distributed client
+          - os: "ubuntu-latest"
+            python: "3.8"
```

Comment:
Do you have a preference on whether the distributed tests should run with the mindeps conda version vs new deps?

Reply:
I opted for mindeps here to make sure that our min versions work in a distributed context, but in reality feel like it would probably be good to test both old and new deps - do you have any opposition to adding another build here for 3.9/3.10?

Reply:
nope that works for me